libs/azure-storage/demos/document_loaders/embed.py

```diff
@@ -0,0 +1,58 @@
+"""Sample showing embedding documents from Azure Blob Storage into Azure Search."""
+
+import os
+import warnings
+
+from azure.identity import DefaultAzureCredential
+from langchain_azure_ai.embeddings import AzureAIEmbeddingsModel
+from langchain_azure_ai.vectorstores import AzureSearch
+from langchain_community.document_loaders import PyPDFLoader
+
+from langchain_azure_storage.document_loaders import AzureBlobStorageLoader
+
+warnings.filterwarnings("ignore")
+
+
+def main() -> None:
+    """Embed documents from Azure Blob Storage into Azure Search."""
+    loader = AzureBlobStorageLoader(
+        account_url=os.environ["AZURE_STORAGE_ACCOUNT_URL"],
+        container_name=os.environ["AZURE_STORAGE_CONTAINER_NAME"],
+        blob_names=[
+            "pdf_test.pdf",
+            "pdf_file2.pdf",
+            "pdf_file3.pdf",
+            "pdf_file4.pdf",
+            "pdf_file5.pdf",
+        ],
```

Comment:
It would be interesting if we could update the loader to support listing from the container itself to show how it can automatically find blobs. I'm thinking we can just set the `prefix` and read from an `AZURE_STORAGE_BLOB_PREFIX` env var that is optional and set the value to `None` if it is not set.